### DISCUSSION TITLE
Encountered an error in the newest version you released, 1.2.14, but have fixed it!

### DISCUSSION THREAD

**Discussion by Linden10** [MacielG1/VodRecovery#14] [Show and tell]
So since you don't have bug report section enabled, I'm here to just report that the newest version you released, 1.2.14, gave me this error:
```
.\Vod_recovery.py
  File "C:\Users\***\Downloads\VodRecovery-1.2.14\Vod_recovery.py", line 215
    f"1) Set Default Video Format \033[94m({get_default_video_format().lstrip(".") or 'mp4'})\033[0m",
                                                                               ^
SyntaxError: f-string: unmatched '('
```

The simple solution was to change the " " in lstrip(".") to ' ' to fix the issue. Here's the fix:
`        f"1) Set Default Video Format \033[94m({get_default_video_format().lstrip('.') or 'mp4'})\033[0m",
`

Hope you don't mind and your welcome! Love all the work you've put into this, it's help me a lot in recovering vods heh! ^_^

**Comment by Linden10** [comment]
Also I like to make a note that I'm having trouble with the TwitchTracker cloudflare since the web browser closes before I the cloudflare verification finishes and reaches the page...should make the timeout a bit longer or such.

I had trouble with streams charts in the last version but no trouble in this version (I think clicking around on the page prevented the web browser from closing).

But with TwitchTracker in this version...yeah not so much.
Have to re-try multiple times just to have the browser window open long enough to verify with cloudflare and reach the website...

**Comment by MacielG1** [comment]
Thanks for the bug report!
Regarding the browser closing issue, have you updated the dependencies? The latest version of SeleniumBase should automatically handle the verification for you. If it still closes, try increasing the `reconnect_time` to a higher value. Let me know if it works

**Reply by Linden10** [reply]
> Thanks for the bug report! Regarding the browser closing issue, have you updated the dependencies? The latest version of SeleniumBase should automatically handle the verification for you. If it still closes, try increasing the `reconnect_time` to a higher value. Let me know if it works

Your welcome and yes I have updated the dependencies before starting the newest version of Vod Recovery 1.2.14! I still had the same issue despite that...

Ah and I did manually upped the reconnect_time in the script though it helped barely any much but after a couple retries it worked.
...Hmm...I wonder if I upped the correct part of the script, oh well I saw you updated to 1.2.15 which is great! Hope this works better heh.

Ah and you opened issues too, awesome!
Thanks!
